### ISSUE TITLE
ARDOP modem not starting

### ISSUE THREAD

**Issue by km4ack** [km4ack/73Linux#122]
per https://groups.io/g/KM4ACK-Pi/topic/109627522

**Comment by km4ack** [comment]
Yes. If you manually install and update Pat Menu, it should work great

**Comment by KD7LXX** [comment]
Thanks for the reply.

Are there different versions of ARDOP. 
Which do I install?

Eric

**Comment by km4ack** [comment]
Use the fork by Peter LaRue. It's ARDOPcf.
https://github.com/pflarue/ardop/tree/develop

Support can be found at https://ardop.groups.io/g/users/topics

**Comment by KD7LXX** [comment]
Thanks!!

When I get home I will load this up.
I really appreciate the help!!
I plan to help the HAMs in our community field ready with WINLINK.
This is credited to your work and willingness to share.

Thanks again!

Eric
